PalmMortgageCalc/PalmMortgageCalc/LoanController.swift

```diff
@@ -53,33 +53,36 @@ class LoanController {
         return totalPayment - interestAmount + loan.additionalPrincipal
     }
     
-    // This function takes in a Loan and returns the cumulative interest paid over the life of the loan.  It was a little tricky for me and I'm not sure it's optimally efficient but it seems to work.  It might also be interesting to add the total number of months, which would be shortened is there is a value for additionalPrincipal
-    func lifeOfLoanAmounts(_ loan: Loan?) -> Double {
+    // This function takes in a Loan and returns the cumulative interest paid over the life of the loan as well as the total number of payments.  Note: to call the result, we need to use the notation as follows:  If we declare a variable of loan1 = lifeOfLoanAmounts(xxxxxxx), we would need to use loan1.totalInterest or loan1.numberPayments
```

Comment:
These are excellent comments you are leaving for a difficult to grasp algorithm. Keep this up!

PS. Option-Command-`/` on a method name will auto-complete with a special documentation comment that will show up whenever you use the method elsewhere.